pkg/component/observability/plutono/plutono.go

```diff
@@ -422,7 +422,9 @@ func (p *plutono) getDashboardConfigMap(ctx context.Context, suffix string) (*co
 			if !p.values.IncludeIstioDashboards {
 				ignorePaths.Insert("istio")
 			}
-			if !p.values.VPNHighAvailabilityEnabled {
+			if p.values.VPNHighAvailabilityEnabled {
+				ignorePaths.Insert("envoy-proxy")
```

Comment:
ignorePath ignores the respective directory to not include for deployment, you should create a new folder called `envoy-proxy` and keep `envoy-proxy-dashboard.json` in that, for below logic to work.
```
				ignorePaths.Insert("envoy-proxy")
```

Reply:
I've done that. 
Github shows it as as `File renamed without changes`.

Reply:
https://github.com/gardener/gardener/pull/10035/commits/c23b2b770e262f680bef24916b14a42324efb524